src/chlorine/ui/connection.cljs

```diff
@@ -46,9 +46,15 @@
 (defn conn-view [cmd]
   (let [div (. js/document (createElement "div"))
         panel (.. js/atom -workspace (addModalPanel #js {:item div}))
-        port-file (-> js/atom .-project .getPaths first
-                      (str "/.shadow-cljs/socket-repl.port"))]
-    (when (existsSync port-file)
+        port-file (->> ["/.shadow-cljs/socket-repl.port"
+                        "/.socket-repl-port"
+                        "/.nrepl-port"]
```

Comment:
Also, you'll have to change these to not have the `/` so they'll work on Windows

Reply:
Adjusted.

I based it off the existing code, so, presumably, it isn't working on Windows currently.